### ISSUE TITLE
Explore convergence strategies for spherical sampling of powders

### ISSUE THREAD

**Issue by ajjackson** [pace-neutrons/Pace-Project-Plan#116]
Blocking #25 

Discussion notes: https://github.com/pace-neutrons/pace-developers/blob/master/euphonic/design/05_powder_averaging_discussion.md

We don't really want our end-users to make decisions about sampling increments in reciprocal-space. We also don't want to specify needlessly dense, expensive sampling meshes.

Experiment with sampling densities to determine whether adaptive strategies are effective and useful. What does the convergence quality/cost look like? Is it important to choose a decent starting point? Can the ideal sampling density be related to properties of the force constant data?

While the ultimate goal here is S(q,w) powder-averaging, the principles are also applicable to band structures, phonon DOS and analogous spin properties. It may be simpler to start with those.

**Comment by ajjackson** [comment]
Somehow this has become a duplicate of #87 . There may have been a fine distinction at one point that no longer applies. I have closed the other one as this Issue has been externally referenced. Copying the notes from that issue here for convenience:

> 
> Depends on #85
> 
> Consider strategies for identifying and accelerating convergence of S(q, ω) with respect to the number of q bins and the number of samples at each surface.
> 
>   -  Should/can we use adaptive sampling to focus on the more difficult areas of the phonon band structure?
>   -  Can we develop an automatic convergence procedure to improve the user experience?
>   -  How accurate/reliable are the results from "typical" settings compared to a fully-converged limit?
> 
> Significant developments in this area could merit an academic publication in their own right, so it would be worth doing a decent literature review first.

**Comment by ajjackson** [comment]
Initial exploration done. Future actions (as discussed in meeting, notes coming soon to project docs) are to:

- Profile powder-averaging script as there seem to be some performance issues #162
- Examine jitter in r-axis #163
- Implement initial API/scripts for a 3-parameter scheme #161 
- Investigate Sobol sampling #164
- Use powder-averaging as a Euphonic-Brille interface test #86
